packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -157,9 +157,9 @@ - (void)imagePickerController:(UIImagePickerController *)picker
       image = [self scaledImage:image maxWidth:maxWidth maxHeight:maxHeight];
     }
 
-    NSData *data = UIImageJPEGRepresentation(image, 1.0);
+    NSData *data = UIImagePNGRepresentation(image);
     NSString *guid = [[NSProcessInfo processInfo] globallyUniqueString];
-    NSString *tmpFile = [NSString stringWithFormat:@"image_picker_%@.jpg", guid];
+    NSString *tmpFile = [NSString stringWithFormat:@"image_picker_%@.png", guid];
```

Comment:
I think that is not good, Force the png mode in any case. mostly in the presence of big images

Reply:
This used to be JPEG with 100% quality. I agree that PNG will probably take a bit more space, but we are doing this to support transparent images. In my opinion this is a good tradeoff.

Do you have any suggestion on how to make this better, while also supporting transparency?

Reply:
I am resolving this conversation so we move on and make progress with this.

Reply:
Updated. Please have a look.

Reply:
@romaluca The latest version of this should address the issue you raised